src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -1042,7 +1042,3 @@ export class TerminalService implements ITerminalService {
 		}
 	}
 }
-
-interface IProfileQuickPickItem extends IQuickPickItem {
-	profile: ITerminalProfile | ITerminalTypeContribution;
-}
```

Comment:
Same here, better to inline within this file. We can move it next to the other interface in this file though,